### DISCUSSION TITLE
Regarding issue 2471 (OneDrive Heic bug)

### DISCUSSION THREAD

**Discussion by per-oestergaard** [abraunegg/onedrive#2589] [General]
Hi.

As commenting on [the issue](https://github.com/abraunegg/onedrive/issues/2471) is locked for me, I'll do an update here and a question.

Info: I have a support case open with Microsoft to see if the HEIC problem can be found/solved.
So far my conclusion is -

- OneDrive App on iOS works as expected (response to POST returns the big size)
- Bug only present when "Live Photo" is used, so it is not directly related to HEIC
- Size from Graph API matches the POST size
- Windows 10 File Explorer shows the small size and download the small size
- OneDrive Web Interface shows the big size
- OneDrive Web Interface downloads the small size
- Using the short-lived download URL returned by the Graph API (the only way to download AFAIK) download the small size
- OneDrive Web Interface never asks for the size from Graph APi, just download using the URL and thus never sees the big size <> small size problem.
- Copying the file with Microsoft Power Automate reduces the file to small
- E.g. the "big" file is "stuck" in OneDrive and cannot be retrieved.

/Per

**Comment by abraunegg** [comment]
@per-oestergaard 
Many thanks for digging into this.

Could you also add these details above to the OneDrive API bug listed here: https://github.com/OneDrive/onedrive-api-docs/issues/1723
